lib/stack_master/change_set.rb

```diff
@@ -42,7 +42,11 @@ def initialize(describe_change_set_response)
     end
 
     def display(io)
-      io.puts "Proposed change set:"
+      io.puts <<-EOL
+
+==========
```

Comment:
Extreme nitpick: can this separator be extended to be longer, either the same length as `Proposed change set:` or perhaps even longer, to enforce the visual separation?

Reply:
Better?
![screen shot 2018-02-21 at 3 19 18 pm](https://user-images.githubusercontent.com/2901882/36462944-a056d05e-171a-11e8-8e7d-735aa0e20ad0.png)

Reply:
❤️